src/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference.c

```diff
@@ -104,26 +103,32 @@ void Update_thrusterPlatformReference(ThrusterPlatformReferenceConfig *configDat
     double FM[3][3];
     tprComputeFinalRotation(r_CM_M, r_TM_F, T_F, FM);
 
-    /*! compute net RW momentum */
-    double vec3[3], hs_B[3], hs_M[3];
-    v3SetZero(hs_B);
-    for (int i=0; i<configData->rwConfigParams.numRW; i++) {
-        v3Scale(configData->rwConfigParams.JsList[i]*rwSpeedMsgIn.wheelSpeeds[i], &configData->rwConfigParams.GsMatrix_B[i*3], vec3);
-        v3Add(hs_B, vec3, hs_B);
+    if (configData->momentumDumping == Yes) {
+        RWSpeedMsgPayload rwSpeedMsgIn = RWSpeedMsg_C_read(&configData->rwSpeedsInMsg);
+
+        /*! compute net RW momentum */
+        double vec3[3], hs_B[3];
```

Comment:
:large_orange_diamond: Variables should each be declared in on their own new line.

Reply:
My bad, I was copying some legacy code here. I fixed it.